### PR TITLE
Landing page, FAQ page and download page update

### LINKED ISSUE #1770: Update Landing Page
Hi, you need to update the landing page based on this design:-
https://www.figma.com/file/JCGeIX9fFrrpZiqE0XzH9s/Homepage?node-id=0%3A1 

It was also mentioned by kristy in this issue: https://github.com/thenewboston-developers/Design/issues/168

If you have any questions, feel free to ask me via tagging me in the comments or via messaging me on discord. Thank you :)

### LINKED ISSUE #1779: Update FAQs Page
Hi, you need to update the FAQ's page; https://thenewboston.com/faq based on the following design.

Here is the Figma link:
https://www.figma.com/file/C22D4Et2I0AIE0n8QwBDMm/FAQ?node-id=152%3A3508
Here is the FAQ content link:
https://docs.google.com/document/d/12DzwJnge7LaVabgtnkIjM9fpuVp1fNS4KdVsndiU4uc/edit?usp=sharing

If you have any query in this design, kindly let me know asap. Thank you!

Note: You can add this task in whichever sprint you want to.

### LINKED ISSUE #1785: Update Download Page
Hi, you need to update the download page as per this design: 
https://www.figma.com/file/dniKTujBGiZmRha4OrtCOT/Download-Wallet?node-id=1%3A10

Design Issue for reference: https://github.com/thenewboston-developers/Design/issues/196

If you need any help or information, kindly reach me out on discord! Thank you.

### PR DESCRIPTION
**Pull request by jamessspanggg**
Fixes #1779
Fixes #1770
Fixes #1785

- [ ] Top nav update
- [x] Home hero update
- [x] Home Value
- [x] Get Started
- [x] FAQ Section
- [x] Footer update
- [x] FAQ Page
- [x] Success/failure toast for feedback form in FAQ page
- [x] Input inline error message UI
- [x] Download page main section
- [ ] Release notes section in download page (need to figure out how to parse the `body` of release)

Things to confirm:
- is it necessary to implement hover to open nav dropdown, the logic complicates everything
- How to implement search for FAQ (currently searching if keyword exists in questions)
- Descriptions for each card in GetStarted section (currently lorem ipsum...)
- Some links that do not exist yet, can search for comments with `// TODO`

References:
- Sending feedback to google sheets: https://nocodeapi.com/integrate-google-sheets-with-your-feedback-form

P.S. apologies for the large PR, most of it is from the svg files

### CODE REVIEW

File: src/apis/faq/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+import {standardHeaders} from 'utils/requests';
+
+export async function sendFeedback(name: string, email: string, response: string) {
+  return axios.post(
+    `https://v1.nocodeapi.com/jamessspanggg/google_sheets/XKBFrUzDbyXOOFNR?tabId=Sheet1&api_key=${process.env.REACT_APP_GOOGLE_SHEET_FAQ_API_KEY}`,
```
Comment (angle943) [inline src/apis/faq/index.ts:6]: @jamessspanggg what is this endpoint? Is it a test endpoint, or the final one that we will be using in PROD?

Comment (jamessspanggg) [reply src/apis/faq/index.ts:6]: this is a test endpoint that we will be using to integrate the contact us submission to google sheets. I think we should create one using our team's email and use that. I stumbled upon this solution via googling, not sure if this is the best solution, also not too sure if we want to store this in google docs or in our BE.

https://nocodeapi.com/integrate-google-sheets-with-your-feedback-form

**Comment by angle943** [comment]
@jamessspanggg i'm currently almost done going through these... Again fantastic work! I'm super amazed by the amount of work you put in into this and all the attention to detail!

**Comment by buckyroberts** [comment]
@angle943 

**Read Documentation** - Learn about thenewboston blockchain.
**Download Wallet** - Interact with the network using our official wallet.
**Join the Community** - Connect with the community to learn, grow, and create.
**Create Proposals** - Earn coins by building projects, gaining skills, and working as a team.
**Complete Bounties** - Work on bug fixes, graphics, and other small tasks to earn coins.
**View Openings** - Apply to be part of the core team.

Screen_Shot_2021-06-07_at_7 10 34_PM

**Review by angle943 — APPROVED** [review]

**Pull request merged**